3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+include(FetchContent)
+
+add_subdirectory(dirent-for-Visual-Studio)
+add_subdirectory(EndianPortable)
+add_subdirectory(Getopt-for-Visual-Studio)
+add_subdirectory(LightPcapNg)
+add_subdirectory(hash-library)
+add_subdirectory(MemPlumber/MemPlumber)
+
+if(PCAPPP_INSTALL)
+  install(
+    TARGETS light_pcapng
+    EXPORT PcapPlusPlusTargets
+    LIBRARY
+    ARCHIVE DESTINATION ${PCAPP_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${PCAPP_INSTALL_LIBDIR}
+    PUBLIC_HEADER DESTINATION ${PCAPP_INSTALL_INCLUDEDIR})
+endif()
```

Comment:
It's quite old but in case light_pcapng have ZSTD enabled and we compile with have a static lib it will not be compiled in the final static libraries (cmake doesn't compile static libraries).

We can make this more proper by including zstd lib in the 3rdParty.
What do you think ?

Reply:
I'm not sure I understand... Do you mean that Zstd won't be compiled into PcapPlusPlus static libs? In that case, how can someone link with the PcapPlusPlus libs? 🤔

Reply:
it should link with pcapplus and libzstd at the same time else the linkder will trig errors, zstd symbols missing

Reply:
We could include the libzstd as a thirdparty to fix this, It could be cleaner IMO

Reply:
> I'm not sure I understand... Do you mean that Zstd won't be compiled into PcapPlusPlus static libs? In that case, how can someone link with the PcapPlusPlus libs? 🤔

I take it back, it was my mistake. We should treat Zstd like any other 3rd party library (for example libpcap, DPDK, PF_RING) - developers should link it when their build their application alongside PcapPlusPlus

Reply:
However - see my comment here: https://github.com/seladb/PcapPlusPlus/pull/944#discussion_r962221762
I think Zstd should be optional and default to `OFF` because I assume most users don't need it.